optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/gizmo/GizmoTestdataEntity.java

```diff
@@ -0,0 +1,46 @@
+package org.optaplanner.core.impl.testdata.domain.gizmo;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.entity.PlanningPin;
+import org.optaplanner.core.api.domain.lookup.PlanningId;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.testdata.domain.TestdataValue;
+
+@PlanningEntity
+public class GizmoTestdataEntity {
+
+    private String id;
+
+    @PlanningVariable
+    public TestdataValue value;
+
+    @PlanningPin
+    public boolean isPinned;
```

Comment:
nitpick: we usually call this field "pinned" in the examples, instead of "isPinned".

Great that this is being tested!